website/screens/principles/component-lifecycle/ComponentLifecycle.tsx

```diff
@@ -0,0 +1,234 @@
+            <DxcBulletedList.Item>
+              The API remains stable, with no breaking changes for at least one
+              month.
+            </DxcBulletedList.Item>
+            <DxcBulletedList.Item>
+              The component implementation remains fairly stable, with no
+              breaking changes for at least one month.
+            </DxcBulletedList.Item>
+          </DxcBulletedList>
+        ),
+      },
+    ],
+  },
+  {
+    title: "Legacy",
+    content: (
+      <>
+        <DxcParagraph>
+          The component is stable but there are plans to replace its
+          functionality with a new component or a combination of new components.
+          The use of legacy components is not discouraged, with the caveat that
+          they will be deprecated and replaced in the next major release.
+        </DxcParagraph>
+      </>
+    ),
+    subSections: [
+      {
+        title: "Implications",
+        content: (
+          <DxcBulletedList>
+            <DxcBulletedList.Item>
+              Documentation exists for the legacy component and includes any
+              alternative components.
+            </DxcBulletedList.Item>
+            <DxcBulletedList.Item>
+              Manual or automated migration paths have been documented and will
+              be code-available as soon as possible.
+            </DxcBulletedList.Item>
+            <DxcBulletedList.Item>
+              The component will be considered as legacy for one release, so
+              users can prepare to its eventual removal.
+            </DxcBulletedList.Item>
+          </DxcBulletedList>
+        ),
+      },
+    ],
+  },
+  {
+    title: "Deprecated",
+    content: (
+      <>
+        <DxcParagraph>
+          The component will be removed in the near future. Users should start
+          avoiding it for new application and existing ones should start
+          considering other approaches.
+        </DxcParagraph>
+      </>
+    ),
+    subSections: [
+      {
+        title: "Implications",
+        content: (
+          <DxcBulletedList>
+            <DxcBulletedList.Item>
+              The component will be removed in the next major release.
+            </DxcBulletedList.Item>
+          </DxcBulletedList>
+        ),
+      },
+    ],
+  },
+];
+
+const ComponentLifecyclePage = () => {
+  return (
+    <DxcFlex direction="column" gap="4rem">
+      <PageHeading>
+        <DxcFlex direction="column" gap="2rem">
+          <DxcHeading
+            level={1}
+            text="Component lifecycle"
+            weight="bold"
+          ></DxcHeading>
+        </DxcFlex>
+      </PageHeading>
+      <QuickNavContainerLayout>
+        <QuickNavContainer
+          sections={sections}
+          startHeadingLevel={2}
+        ></QuickNavContainer>
+      </QuickNavContainerLayout>
+      <DocFooter githubLink="https://github.com/dxc-technology/halstack-react/blob/master/website/screens/principles/color/ColorPage.tsx" />
+    </DxcFlex>
```

Comment:
Please change the `githubLink` to the `ComponentLifecycle` one.

Reply:
Updated!